apps/sms/test/unit/drafts_test.js

```diff
@@ -105,72 +106,60 @@ suite('Drafts', function() {
     mocksHelper.suiteTeardown();
   });
 
-  suite('Drafts() >', function() {
-
-    test('Draft', function() {
```

Comment:
The developer that was in charge liked such tests to check the API was there -- I never really liked them so I'm fine removing it.